domain/provider.go

```diff
@@ -7,17 +7,18 @@ import (
 )
 
 const (
-	ProviderTypeBigQuery  = "bigquery"
-	ProviderTypeMetabase  = "metabase"
-	ProviderTypeGrafana   = "grafana"
-	ProviderTypeTableau   = "tableau"
-	ProviderTypeGCloudIAM = "gcloud_iam"
-	ProviderTypeNoOp      = "noop"
-	ProviderTypeGCS       = "gcs"
-	ProviderTypePolicyTag = "dataplex"
-	ProviderTypeShield    = "shield"
-	ProviderTypeGitlab    = "gitlab"
-	ProviderTypeGate      = "gate"
+	ProviderTypeAliCloudIAM = "alicloud_iam"
```

Comment:
`alicloud_ram` would be the suitable one instead of `alicloud_iam`
```suggestion
	ProviderTypeAliCloudIAM = "alicloud_ram"
```

Reply:
please also change the plugin name accordingly

Reply:
Resolved: https://github.com/goto/guardian/pull/190/commits/6bd8fdffe0d3723f936bc84ddaf326d111b06d9b